Code/Editor/Plugins/ComponentEntityEditorPlugin/UI/Outliner/OutlinerListModel.cpp

```diff
@@ -605,6 +605,13 @@ bool OutlinerListModel::setData(const QModelIndex& index, const QVariant& value,
 
                     if (oldName != newName)
                     {
+                        QRegExp rx("[_a-zA-Z0-9-\\s]+");
+                        if (!rx.exactMatch(newName.c_str()))
+                        {
+                            QMessageBox::information(NULL, "Title", "Special characters are not allowed", QMessageBox::Yes | QMessageBox::No, QMessageBox::Yes);
```

Comment:
Please make the same parent change here.

Reply:
Should this be a yes/no dialog? There is no question in the message and there is no logic branching from the result of yes or no